src/level_four/core/tasks/send_transaction_tasks.c

```diff
@@ -139,13 +133,12 @@ void send_transaction_tasks()
         char display[225] = {0};
         uint8_t precision = get_floating_precision(value, SATOSHI_PER_BTC);
         snprintf(display, sizeof(display), "Receiver #%d\nSend %0.*f\n%s", var_send_transaction_data.transaction_confirmation_list_index + 1, precision, valueToDisplay, get_coin_symbol(BYTE_ARRAY_TO_UINT32(var_send_transaction_data.transaction_metadata.coin_index), var_send_transaction_data.transaction_metadata.network_chain_id));
-        confirm_scr_init(display);
+        ui_scrollable_page(ui_heading_confirm_action, display, MENU_SCROLL_HORIZONTAL, false);
```

Comment:
```suggestion
        ui_scrollable_page(ui_heading_verify_amount, display, MENU_SCROLL_HORIZONTAL, false);
```

Reply:
This is followed in ETH, NEAR, and SOLANA in this PR, so should be followed as same here for consistency.